admin/data/sql/migrations/086.do.create-pupil-census-table.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE [mtc_admin].[pupilCensus] (
+  id        INT IDENTITY (1, 1) NOT NULL,
+  urlSlug [uniqueidentifier] NOT NULL DEFAULT (newid()),
+  name      [nvarchar](MAX) NOT NULL,
+  blobFileName [nvarchar](300) NOT NULL,
+  createdAt DATETIMEOFFSET(3)   NOT NULL DEFAULT GETUTCDATE(),
+  updatedAt DATETIMEOFFSET(3)   NOT NULL DEFAULT GETUTCDATE(),
+  version rowversion,
+  jobStatus_id int NOT NULL,
+  isDeleted [BIT] NOT NULL DEFAULT 0,
```

Comment:
What's `isDeleted` for?

Reply:
@jon-shipley there is always going to be only 1 pupil census active, which means any previous pupil census will be flagged as deleted when the user performs the delete action in the UI.
However this is one point that is yet to be discussed.
If you have any different ideas let me know.